expanse-app/src/component/AddNew.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react'
+import { useDispatch,useSelector } from 'react-redux';
+import { TransactionAction } from '../store/transactionSlice';
+const AddNew = () => {
+    const [amout,setAmout]=useState('');
+    const [date,setDate]=useState();
+    const [note,setNote]=useState('');
+    const [type,setType]= useState('income');
```

Comment:
Nên dùng 1 object thay vì từng property như thế này
```suggestion
    const [formValue,setFormValue]=useState({amout: 0, date: '', note, ''});
```